README.md

```diff
@@ -1,8 +1,261 @@
+    strong consensus *against* that position outside of the subteam. Subteam
+    members use their best judgment in taking this step, and the FCP itself
+    ensures there is ample time and notification for stakeholders to push back
+    if it is made prematurely.
+    - For RFCs with lengthy discussion, the motion to FCP is usually preceded by
+      a *summary comment* trying to lay out the current state of the discussion
+      and major tradeoffs/points of disagreement.
+    - Before actually entering FCP, *all* members of the subteam must sign off;
+    this is often the point at which many subteam members first review the RFC
+    in full depth.
+  - The FCP lasts ten calendar days, so that it is open for at least 5 business
+    days. It is also advertised widely,
+    e.g. in [This Week in Amethyst](https://amethyst.rs/). This way all
+    stakeholders have a chance to lodge any final objections before a decision
+    is reached.
+  - In most cases, the FCP period is quiet, and the RFC is either merged or
+    closed. However, sometimes substantial new arguments or ideas are raised,
+    the FCP is canceled, and the RFC goes back into development mode.
+
+## The RFC life-cycle
+[The RFC life-cycle]: #the-rfc-life-cycle
+
+Once an RFC becomes "active" then authors may implement it and submit the
+feature as a pull request to the Amethyst repo. Being "active" is not a rubber
+stamp, and in particular still does not mean the feature will ultimately be
+merged; it does mean that in principle all the major stakeholders have agreed
+to the feature and are amenable to merging it.
+
+Furthermore, the fact that a given RFC has been accepted and is "active"
+implies nothing about what priority is assigned to its implementation, nor does
+it imply anything about whether an Amethyst developer has been assigned the task of
+implementing the feature. While it is not *necessary* that the author of the
+RFC also write the implementation, it is by far the most effective way to see
+an RFC through to completion: authors should not expect that other project
+developers will take on responsibility for implementing their accepted feature.
+
+Modifications to "active" RFCs can be done in follow-up pull requests. We
+strive to write each RFC in a manner that it will reflect the final design of
+the feature; but the nature of the process means that we cannot expect every
+merged RFC to actually reflect what the end result will be at the time of the
+next major release.
+
+In general, once accepted, RFCs should not be substantially changed. Only very
+minor changes should be submitted as amendments. More substantial changes
+should be new RFCs, with a note added to the original RFC. Exactly what counts
+as a "very minor change" is up to the sub-team to decide; check
+[Sub-team specific guidelines] for more details.
+
+
+## Reviewing RFCs
+[Reviewing RFCs]: #reviewing-rfcs
+
+While the RFC pull request is up, the sub-team may schedule meetings with the
```

Comment:
That's actually contradictory to our org policy, which describes that the core team approves RFCs. Right now the sub-teams are a bit small to do this, not sure how we want to handle it. I think the sub-teams deserve to have special weight, but I'm not sure how to declare that formally.

Reply:
Can you share a link to the org policy, as well as a description of the responsibilities of the Core Team?

Reply:
or anyone on the @amethyst/core presumably that can provide those?

Reply:
If that is an org policy, I do think it deserves to scrapped since it was made with the old structure in mind. I assume @torkleyy is talking about some things mentioned in [this document](https://github.com/amethyst/amethyst/blob/master/docs/CONTRIBUTING.md) but I honestly feel like the sub teams take priority here.

Reply:
Let's leave it like this, and we'll handle it on a case-by-case basis until the sub-teams are large enough